frontend/src/components/forms/CareerDiffModal.module.css

```
@@ -243,3 +254,58 @@
+.proofreadItem {
+  border-left: 2px solid rgba(37, 99, 235, 0.45);
+  padding-left: 0.45rem;
+}
+
+.proofreadMessage {
+  margin: 0;
+  font-size: 0.74rem;
+  color: var(--text-primary, #1f2937);
+  line-height: 1.4;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)